scripts/start-story/main.go

```diff
@@ -120,7 +121,7 @@ func run() error {
 	}
 	finish()
 
-	if meta.JiraIssue.Fields.Status.Name == "To Do" || meta.JiraIssue.Fields.Status.Name == "Pending" {
+	if meta.JiraIssue.Fields.Status.Name == wh.JiraStatusTodo || meta.JiraIssue.Fields.Status.Name == wh.JiraStatusPending {
 		finish = wc.PrintStart("Updating jira issue to In Progress")
 		if err := wh.UpdateJiraStatus(jiraClient, meta.JiraIssue, "In Progress"); err != nil {
```

Comment:
You'll want to use a constant for "In Progress" too.

Reply:
Woops, thanks for catching!